src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java

```diff
@@ -90,11 +90,9 @@ static void disableIllegalReflectiveAccessWarning() {
 
   private static void initializeClientVersionFromManifest() {
     /*
-     * Get JDBC version numbers from version.properties in snowflake-jdbc
+     * Get JDBC version numbers from static string in snowflake-jdbc
```

Comment:
NIT: It looks `versionResourceBundleManager` is not used anymore. It may be removed too.